osmaxx/excerptexport/models/excerpt.py

```diff
@@ -1,24 +1,34 @@
-from django.db import models
 from django.contrib.auth.models import User
+from django.contrib.gis.db import models
 from django.utils.translation import gettext_lazy as _
 
 
 class Excerpt(models.Model):
     name = models.CharField(max_length=128, verbose_name=_('name'))
     is_public = models.BooleanField(default=False, verbose_name=_('is public'))
     is_active = models.BooleanField(default=True, verbose_name=_('is active'))
-
     owner = models.ForeignKey(User, related_name='excerpts', verbose_name=_('owner'))
-    bounding_geometry = models.OneToOneField('BoundingGeometry', verbose_name=_('bounding geometry'))
+    bounding_geometry = models.MultiPolygonField(verbose_name=_('bounding geometry'), null=True)
+    country = models.ForeignKey('countries.Country', verbose_name=_('Country'), null=True)
```

Comment:
`on_delete=RESTRICT` would not hamper consistency. It would mean that the DB would refuse to delete a country as long as it is referenced by at least one excerpt.

`CASCADE` means that deleting a country will implicitly delete all excerpts referencing it.